packages/feedback/src/core/createMainStyles.ts

```diff
@@ -33,6 +33,8 @@ function getThemedCssVariables(theme: FeedbackTheme): string {
 
   --form-border-radius: ${theme.formBorderRadius};
   --form-content-border-radius: ${theme.formContentBorderRadius};
+
+  --crop-foreground: ${theme.cropForeground};
```

Comment:
imo, we don't allow theming of this button and use submitButton theme instead. This forces the UI to be a bit more consistent. I think it's ok to keep the CSS variable.

```suggestion

  --crop-foreground: ${theme.submitForeground};
```

Reply:
the crop "buttons" I'm referring to are the 4 corners that you can drag to crop, I'll rename them so it's more obvious

Reply:
Ah can you post a screenshot in the PR and maybe consult with @Jesse-Box? I think similar idea applies even though this isn't strictly a button, we want to maintain some sort of uniform theming so there aren't too many one-off colors.